ci/src/main/scala/org/typelevel/sbt/TypelevelCiPlugin.scala

```diff
@@ -0,0 +1,29 @@
+package org.typelevel.sbt
+
+import sbt._
+import sbtghactions.GenerativePlugin
+import sbtghactions.GitHubActionsPlugin
+import sbtghactions.GenerativePlugin.autoImport._
+import com.typesafe.tools.mima.plugin.MimaPlugin
+
+object TypelevelCiPlugin extends AutoPlugin {
+
+  override def requires = GitHubActionsPlugin && GenerativePlugin && MimaPlugin
+  override def trigger = allRequirements
+
+  override def buildSettings =
+    addCommandAlias(
+      "ci",
```

Comment:
Thought exercise: how would we make this open to extension for projects that run additional checks?  This has frustrated me in other plugins that use aliases.

Reply:
Oh, `replaceCommandAlias`?

Reply:
Yep, got that here:
https://github.com/typelevel/sbt-typelevel/blob/f277ca92734b427e0db106d43fbdec8a6948eddb/kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala#L11-L15

Reply:
So much else is structured, and we manipulate aliases as semi-colon delimited strings.  But I guess we inherit that from SBT's model.  This is probably the best we can do.